### ISSUE TITLE
Fix CS issues for the tests

### ISSUE THREAD

**Issue by andizer** [Yoast/wpseo-woocommerce#408]
I've excluded the tests directory (See https://github.com/Yoast/wpseo-woocommerce/pull/405/files#diff-325a505e494aa8078473d0b71ad82200R20) from being checked for the PHP CS, this is because the 'plugin-grunt-tasks' dependency has been updated causing a lot of new CS errors. Because of fixing these errors were out of scope I decided to exclude the tests directory.

When we stop supporting PHP 5.2 we can remove the exclusion and maybe then fix possible CS errors.

**Comment by IreneStr** [comment]
The error output:
```
Running "shell:phpcs" (shell) task
.E..E....E......E. 18 / 18 (100%)



FILE: tests/doubles/schema-double.php
----------------------------------------------------------------------
FOUND 6 ERRORS AFFECTING 2 LINES
----------------------------------------------------------------------
 3 | ERROR | "namespace" keyword is not present in PHP version 5.2 or
   |       | earlier
   |       | (PHPCompatibility.Keywords.NewKeywords.t_namespaceFound)
 3 | ERROR | the \ operator (for namespaces) is not present in PHP
   |       | version 5.2 or earlier
   |       | (PHPCompatibility.LanguageConstructs.NewLanguageConstructs.t_ns_separatorFound)
 3 | ERROR | the \ operator (for namespaces) is not present in PHP
   |       | version 5.2 or earlier
   |       | (PHPCompatibility.LanguageConstructs.NewLanguageConstructs.t_ns_separatorFound)
 3 | ERROR | the \ operator (for namespaces) is not present in PHP
   |       | version 5.2 or earlier
   |       | (PHPCompatibility.LanguageConstructs.NewLanguageConstructs.t_ns_separatorFound)
 3 | ERROR | the \ operator (for namespaces) is not present in PHP
   |       | version 5.2 or earlier
   |       | (PHPCompatibility.LanguageConstructs.NewLanguageConstructs.t_ns_separatorFound)
 5 | ERROR | "use" keyword (for traits/namespaces/anonymous
   |       | functions) is not present in PHP version 5.2 or
   |       | earlier
   |       | (PHPCompatibility.Keywords.NewKeywords.t_useFound)
----------------------------------------------------------------------


FILE: tests/bootstrap.php
----------------------------------------------------------------------
FOUND 1 ERROR AFFECTING 1 LINE
----------------------------------------------------------------------
 27 | ERROR | __DIR__ magic constant is not present in PHP version
    |       | 5.2 or earlier
    |       | (PHPCompatibility.Keywords.NewKeywords.t_dirFound)
----------------------------------------------------------------------


FILE: tests/TestCase.php
----------------------------------------------------------------------
FOUND 25 ERRORS AFFECTING 16 LINES
----------------------------------------------------------------------
  3 | ERROR | "namespace" keyword is not present in PHP version 5.2
    |       | or earlier
    |       | (PHPCompatibility.Keywords.NewKeywords.t_namespaceFound)
  3 | ERROR | the \ operator (for namespaces) is not present in PHP
    |       | version 5.2 or earlier
    |       | (PHPCompatibility.LanguageConstructs.NewLanguageConstructs.t_ns_separatorFound)
  3 | ERROR | the \ operator (for namespaces) is not present in PHP
    |       | version 5.2 or earlier
    |       | (PHPCompatibility.LanguageConstructs.NewLanguageConstructs.t_ns_separatorFound)
  3 | ERROR | the \ operator (for namespaces) is not present in PHP
    |       | version 5.2 or earlier
    |       | (PHPCompatibility.LanguageConstructs.NewLanguageConstructs.t_ns_separatorFound)
  5 | ERROR | "use" keyword (for traits/namespaces/anonymous
    |       | functions) is not present in PHP version 5.2 or
    |       | earlier
    |       | (PHPCompatibility.Keywords.NewKeywords.t_useFound)
  6 | ERROR | "use" keyword (for traits/namespaces/anonymous
    |       | functions) is not present in PHP version 5.2 or
    |       | earlier
    |       | (PHPCompatibility.Keywords.NewKeywords.t_useFound)
  6 | ERROR | the \ operator (for namespaces) is not present in PHP
    |       | version 5.2 or earlier
    |       | (PHPCompatibility.LanguageConstructs.NewLanguageConstructs.t_ns_separatorFound)
 19 | ERROR | the \ operator (for namespaces) is not present in PHP
    |       | version 5.2 or earlier
    |       | (PHPCompatibility.LanguageConstructs.NewLanguageConstructs.t_ns_separatorFound)
 21 | ERROR | the \ operator (for namespaces) is not present in PHP
    |       | version 5.2 or earlier
    |       | (PHPCompatibility.LanguageConstructs.NewLanguageConstructs.t_ns_separatorFound)
 21 | ERROR | the \ operator (for namespaces) is not present in PHP
    |       | version 5.2 or earlier
    |       | (PHPCompatibility.LanguageConstructs.NewLanguageConstructs.t_ns_separatorFound)
 22 | ERROR | Short array syntax (open) is available since 5.4
    |       | (PHPCompatibility.Syntax.NewShortArray.Found)
 35 | ERROR | Closures / anonymous functions are not available in PHP
    |       | 5.2 or earlier
    |       | (PHPCompatibility.FunctionDeclarations.NewClosure.Found)
 40 | ERROR | Closures / anonymous functions are not available in PHP
    |       | 5.2 or earlier
    |       | (PHPCompatibility.FunctionDeclarations.NewClosure.Found)
 43 | ERROR | Short array syntax (close) is available since 5.4
    |       | (PHPCompatibility.Syntax.NewShortArray.Found)
 46 | ERROR | the \ operator (for namespaces) is not present in PHP
    |       | version 5.2 or earlier
    |       | (PHPCompatibility.LanguageConstructs.NewLanguageConstructs.t_ns_separatorFound)
 46 | ERROR | the \ operator (for namespaces) is not present in PHP
    |       | version 5.2 or earlier
    |       | (PHPCompatibility.LanguageConstructs.NewLanguageConstructs.t_ns_separatorFound)
 48 | ERROR | the \ operator (for namespaces) is not present in PHP
    |       | version 5.2 or earlier
    |       | (PHPCompatibility.LanguageConstructs.NewLanguageConstructs.t_ns_separatorFound)
 49 | ERROR | Short array syntax (open) is available since 5.4
    |       | (PHPCompatibility.Syntax.NewShortArray.Found)
 49 | ERROR | Short array syntax (close) is available since 5.4
    |       | (PHPCompatibility.Syntax.NewShortArray.Found)
 51 | ERROR | the \ operator (for namespaces) is not present in PHP
    |       | version 5.2 or earlier
    |       | (PHPCompatibility.LanguageConstructs.NewLanguageConstructs.t_ns_separatorFound)
 51 | ERROR | the \ operator (for namespaces) is not present in PHP
    |       | version 5.2 or earlier
    |       | (PHPCompatibility.LanguageConstructs.NewLanguageConstructs.t_ns_separatorFound)
 53 | ERROR | the \ operator (for namespaces) is not present in PHP
    |       | version 5.2 or earlier
    |       | (PHPCompatibility.LanguageConstructs.NewLanguageConstructs.t_ns_separatorFound)
 54 | ERROR | Short array syntax (open) is available since 5.4
    |       | (PHPCompatibility.Syntax.NewShortArray.Found)
 54 | ERROR | Short array syntax (close) is available since 5.4
    |       | (PHPCompatibility.Syntax.NewShortArray.Found)
 61 | ERROR | the \ operator (for namespaces) is not present in PHP
    |       | version 5.2 or earlier
    |       | (PHPCompatibility.LanguageConstructs.NewLanguageConstructs.t_ns_separatorFound)
----------------------------------------------------------------------


FILE: tests/classes/schema-test.php
----------------------------------------------------------------------
FOUND 75 ERRORS AFFECTING 51 LINES
----------------------------------------------------------------------
   3 | ERROR | "namespace" keyword is not present in PHP version 5.2
     |       | or earlier
     |       | (PHPCompatibility.Keywords.NewKeywords.t_namespaceFound)
   3 | ERROR | the \ operator (for namespaces) is not present in PHP
     |       | version 5.2 or earlier
     |       | (PHPCompatibility.LanguageConstructs.NewLanguageConstructs.t_ns_separatorFound)
   3 | ERROR | the \ operator (for namespaces) is not present in PHP
     |       | version 5.2 or earlier
     |       | (PHPCompatibility.LanguageConstructs.NewLanguageConstructs.t_ns_separatorFound)
   3 | ERROR | the \ operator (for namespaces) is not present in PHP
     |       | version 5.2 or earlier
     |       | (PHPCompatibility.LanguageConstructs.NewLanguageConstructs.t_ns_separatorFound)
   3 | ERROR | the \ operator (for namespaces) is not present in PHP
     |       | version 5.2 or earlier
     |       | (PHPCompatibility.LanguageConstructs.NewLanguageConstructs.t_ns_separatorFound)
   5 | ERROR | "use" keyword (for traits/namespaces/anonymous
     |       | functions) is not present in PHP version 5.2 or
     |       | earlier
     |       | (PHPCompatibility.Keywords.NewKeywords.t_useFound)
   5 | ERROR | the \ operator (for namespaces) is not present in PHP
     |       | version 5.2 or earlier
     |       | (PHPCompatibility.LanguageConstructs.NewLanguageConstructs.t_ns_separatorFound)
   6 | ERROR | "use" keyword (for traits/namespaces/anonymous
     |       | functions) is not present in PHP version 5.2 or
     |       | earlier
     |       | (PHPCompatibility.Keywords.NewKeywords.t_useFound)
   6 | ERROR | the \ operator (for namespaces) is not present in PHP
     |       | version 5.2 or earlier
     |       | (PHPCompatibility.LanguageConstructs.NewLanguageConstructs.t_ns_separatorFound)
   6 | ERROR | the \ operator (for namespaces) is not present in PHP
     |       | version 5.2 or earlier
     |       | (PHPCompatibility.LanguageConstructs.NewLanguageConstructs.t_ns_separatorFound)
   7 | ERROR | "use" keyword (for traits/namespaces/anonymous
     |       | functions) is not present in PHP version 5.2 or
     |       | earlier
     |       | (PHPCompatibility.Keywords.NewKeywords.t_useFound)
   8 | ERROR | "use" keyword (for traits/namespaces/anonymous
     |       | functions) is not present in PHP version 5.2 or
     |       | earlier
     |       | (PHPCompatibility.Keywords.NewKeywords.t_useFound)
   9 | ERROR | "use" keyword (for traits/namespaces/anonymous
     |       | functions) is not present in PHP version 5.2 or
     |       | earlier
     |       | (PHPCompatibility.Keywords.NewKeywords.t_useFound)
   9 | ERROR | the \ operator (for namespaces) is not present in PHP
     |       | version 5.2 or earlier
     |       | (PHPCompatibility.LanguageConstructs.NewLanguageConstructs.t_ns_separatorFound)
   9 | ERROR | the \ operator (for namespaces) is not present in PHP
     |       | version 5.2 or earlier
     |       | (PHPCompatibility.LanguageConstructs.NewLanguageConstructs.t_ns_separatorFound)
   9 | ERROR | the \ operator (for namespaces) is not present in PHP
     |       | version 5.2 or earlier
     |       | (PHPCompatibility.LanguageConstructs.NewLanguageConstructs.t_ns_separatorFound)
   9 | ERROR | the \ operator (for namespaces) is not present in PHP
     |       | version 5.2 or earlier
     |       | (PHPCompatibility.LanguageConstructs.NewLanguageConstructs.t_ns_separatorFound)
   9 | ERROR | the \ operator (for namespaces) is not present in PHP
     |       | version 5.2 or earlier
     |       | (PHPCompatibility.LanguageConstructs.NewLanguageConstructs.t_ns_separatorFound)
  10 | ERROR | "use" keyword (for traits/namespaces/anonymous
     |       | functions) is not present in PHP version 5.2 or
     |       | earlier
     |       | (PHPCompatibility.Keywords.NewKeywords.t_useFound)
  10 | ERROR | the \ operator (for namespaces) is not present in PHP
     |       | version 5.2 or earlier
     |       | (PHPCompatibility.LanguageConstructs.NewLanguageConstructs.t_ns_separatorFound)
  10 | ERROR | the \ operator (for namespaces) is not present in PHP
     |       | version 5.2 or earlier
     |       | (PHPCompatibility.LanguageConstructs.NewLanguageConstructs.t_ns_separatorFound)
  10 | ERROR | the \ operator (for namespaces) is not present in PHP
     |       | version 5.2 or earlier
     |       | (PHPCompatibility.LanguageConstructs.NewLanguageConstructs.t_ns_separatorFound)
  10 | ERROR | the \ operator (for namespaces) is not present in PHP
     |       | version 5.2 or earlier
     |       | (PHPCompatibility.LanguageConstructs.NewLanguageConstructs.t_ns_separatorFound)
  23 | ERROR | the \ operator (for namespaces) is not present in PHP
     |       | version 5.2 or earlier
     |       | (PHPCompatibility.LanguageConstructs.NewLanguageConstructs.t_ns_separatorFound)
  23 | ERROR | the \ operator (for namespaces) is not present in PHP
     |       | version 5.2 or earlier
     |       | (PHPCompatibility.LanguageConstructs.NewLanguageConstructs.t_ns_separatorFound)
  28 | ERROR | the \ operator (for namespaces) is not present in PHP
     |       | version 5.2 or earlier
     |       | (PHPCompatibility.LanguageConstructs.NewLanguageConstructs.t_ns_separatorFound)
  28 | ERROR | the \ operator (for namespaces) is not present in PHP
     |       | version 5.2 or earlier
     |       | (PHPCompatibility.LanguageConstructs.NewLanguageConstructs.t_ns_separatorFound)
  56 | ERROR | Short array syntax (open) is available since 5.4
     |       | (PHPCompatibility.Syntax.NewShortArray.Found)
  57 | ERROR | Short array syntax (open) is available since 5.4
     |       | (PHPCompatibility.Syntax.NewShortArray.Found)
  61 | ERROR | Short array syntax (close) is available since 5.4
     |       | (PHPCompatibility.Syntax.NewShortArray.Found)
  62 | ERROR | Short array syntax (close) is available since 5.4
     |       | (PHPCompatibility.Syntax.NewShortArray.Found)
  66 | ERROR | the \ operator (for namespaces) is not present in PHP
     |       | version 5.2 or earlier
     |       | (PHPCompatibility.LanguageConstructs.NewLanguageConstructs.t_ns_separatorFound)
  67 | ERROR | Short array syntax (open) is available since 5.4
     |       | (PHPCompatibility.Syntax.NewShortArray.Found)
  69 | ERROR | Short array syntax (close) is available since 5.4
     |       | (PHPCompatibility.Syntax.NewShortArray.Found)
  72 | ERROR | The magic class constant ClassName::class was not
     |       | available in PHP 5.4 or earlier
     |       | (PHPCompatibility.Constants.NewMagicClassConstant.Found)
  74 | ERROR | Short array syntax (open) is available since 5.4
     |       | (PHPCompatibility.Syntax.NewShortArray.Found)
  74 | ERROR | Short array syntax (close) is available since 5.4
     |       | (PHPCompatibility.Syntax.NewShortArray.Found)
  75 | ERROR | Short array syntax (open) is available since 5.4
     |       | (PHPCompatibility.Syntax.NewShortArray.Found)
  84 | ERROR | Short array syntax (close) is available since 5.4
     |       | (PHPCompatibility.Syntax.NewShortArray.Found)
  86 | ERROR | Short array syntax (open) is available since 5.4
     |       | (PHPCompatibility.Syntax.NewShortArray.Found)
  94 | ERROR | Short array syntax (open) is available since 5.4
     |       | (PHPCompatibility.Syntax.NewShortArray.Found)
  95 | ERROR | Short array syntax (open) is available since 5.4
     |       | (PHPCompatibility.Syntax.NewShortArray.Found)
  99 | ERROR | Short array syntax (open) is available since 5.4
     |       | (PHPCompatibility.Syntax.NewShortArray.Found)
 103 | ERROR | Short array syntax (close) is available since 5.4
     |       | (PHPCompatibility.Syntax.NewShortArray.Found)
 104 | ERROR | Short array syntax (close) is available since 5.4
     |       | (PHPCompatibility.Syntax.NewShortArray.Found)
 105 | ERROR | Short array syntax (close) is available since 5.4
     |       | (PHPCompatibility.Syntax.NewShortArray.Found)
 106 | ERROR | Short array syntax (close) is available since 5.4
     |       | (PHPCompatibility.Syntax.NewShortArray.Found)
 108 | ERROR | Short array syntax (open) is available since 5.4
     |       | (PHPCompatibility.Syntax.NewShortArray.Found)
 113 | ERROR | Short array syntax (open) is available since 5.4
     |       | (PHPCompatibility.Syntax.NewShortArray.Found)
 113 | ERROR | Short array syntax (close) is available since 5.4
     |       | (PHPCompatibility.Syntax.NewShortArray.Found)
 116 | ERROR | Short array syntax (open) is available since 5.4
     |       | (PHPCompatibility.Syntax.NewShortArray.Found)
 117 | ERROR | Short array syntax (open) is available since 5.4
     |       | (PHPCompatibility.Syntax.NewShortArray.Found)
 121 | ERROR | Short array syntax (open) is available since 5.4
     |       | (PHPCompatibility.Syntax.NewShortArray.Found)
 123 | ERROR | Short array syntax (close) is available since 5.4
     |       | (PHPCompatibility.Syntax.NewShortArray.Found)
 124 | ERROR | Short array syntax (close) is available since 5.4
     |       | (PHPCompatibility.Syntax.NewShortArray.Found)
 125 | ERROR | Short array syntax (close) is available since 5.4
     |       | (PHPCompatibility.Syntax.NewShortArray.Found)
 126 | ERROR | Short array syntax (open) is available since 5.4
     |       | (PHPCompatibility.Syntax.NewShortArray.Found)
 126 | ERROR | Short array syntax (close) is available since 5.4
     |       | (PHPCompatibility.Syntax.NewShortArray.Found)
 127 | ERROR | Short array syntax (open) is available since 5.4
     |       | (PHPCompatibility.Syntax.NewShortArray.Found)
 127 | ERROR | Short array syntax (close) is available since 5.4
     |       | (PHPCompatibility.Syntax.NewShortArray.Found)
 128 | ERROR | Short array syntax (open) is available since 5.4
     |       | (PHPCompatibility.Syntax.NewShortArray.Found)
 131 | ERROR | Short array syntax (close) is available since 5.4
     |       | (PHPCompatibility.Syntax.NewShortArray.Found)
 132 | ERROR | Short array syntax (open) is available since 5.4
     |       | (PHPCompatibility.Syntax.NewShortArray.Found)
 135 | ERROR | Short array syntax (close) is available since 5.4
     |       | (PHPCompatibility.Syntax.NewShortArray.Found)
 136 | ERROR | Short array syntax (close) is available since 5.4
     |       | (PHPCompatibility.Syntax.NewShortArray.Found)
 155 | ERROR | the \ operator (for namespaces) is not present in PHP
     |       | version 5.2 or earlier
     |       | (PHPCompatibility.LanguageConstructs.NewLanguageConstructs.t_ns_separatorFound)
 156 | ERROR | the \ operator (for namespaces) is not present in PHP
     |       | version 5.2 or earlier
     |       | (PHPCompatibility.LanguageConstructs.NewLanguageConstructs.t_ns_separatorFound)
 177 | ERROR | the \ operator (for namespaces) is not present in PHP
     |       | version 5.2 or earlier
     |       | (PHPCompatibility.LanguageConstructs.NewLanguageConstructs.t_ns_separatorFound)
 181 | ERROR | the \ operator (for namespaces) is not present in PHP
     |       | version 5.2 or earlier
     |       | (PHPCompatibility.LanguageConstructs.NewLanguageConstructs.t_ns_separatorFound)
 185 | ERROR | Short array syntax (open) is available since 5.4
     |       | (PHPCompatibility.Syntax.NewShortArray.Found)
 188 | ERROR | Short array syntax (close) is available since 5.4
     |       | (PHPCompatibility.Syntax.NewShortArray.Found)
 209 | ERROR | the \ operator (for namespaces) is not present in PHP
     |       | version 5.2 or earlier
     |       | (PHPCompatibility.LanguageConstructs.NewLanguageConstructs.t_ns_separatorFound)
 210 | ERROR | the \ operator (for namespaces) is not present in PHP
     |       | version 5.2 or earlier
     |       | (PHPCompatibility.LanguageConstructs.NewLanguageConstructs.t_ns_separatorFound)
 210 | ERROR | Short array syntax (open) is available since 5.4
     |       | (PHPCompatibility.Syntax.NewShortArray.Found)
 210 | ERROR | Short array syntax (close) is available since 5.4
     |       | (PHPCompatibility.Syntax.NewShortArray.Found)
----------------------------------------------------------------------

Time: 2.22 secs; Memory: 14MB
```

**Comment by IreneStr** [comment]
@jrfnl Some of the above errors assume that we use PHP <=5.2, for example: 
``` 
 3 | ERROR | "namespace" keyword is not present in PHP version 5.2 or
   |       | earlier
   |       | (PHPCompatibility.Keywords.NewKeywords.t_namespaceFound)
```  

while other are about PHP > 5.4, for example: 
``` 
  74 | ERROR | Short array syntax (close) is available since 5.4
     |       | (PHPCompatibility.Syntax.NewShortArray.Found)
```  

How does this work, and where does PHPCS find which PHP version we're using?
